public/bz_support.js

```diff
@@ -28,6 +28,8 @@ function bzRetainedInfoSetup() {
       ta.checked = (value == ta.value) ? true : false;
     else if(ta.tagName == "INPUT" || ta.tagName == "TEXTAREA")
       ta.value = value;
+    else if(ta.tagName == "INPUT" && ta.getAttribute("type") == "button")
```

Comment:
you'll have to move this up, the condition before it always triggers since it is more general